src/main/java/io/pakland/mdas/githubstats/domain/model/UserReviewAggregation.java

```diff
@@ -0,0 +1,22 @@
+package io.pakland.mdas.githubstats.domain.model;
+
+import java.util.List;
+
+public class UserReviewAggregation {
+
+    private final int commentLengthSum;
+
+    private UserReviewAggregation(int commentLengthSum) {
+        this.commentLengthSum = commentLengthSum;
+    }
+
+    public static UserReviewAggregation aggregate(List<UserReview> userReviews) {
+        int commentLengthSum = userReviews.stream().mapToInt(UserReview::sumCommentLength).sum();
+        return new UserReviewAggregation(commentLengthSum);
+    }
```

Comment:
Does it make sense to be able to have a regular constructor and a named constructor?

Reply:
I don't quite see the need for it as we'll probably instantiate only by performing the aggregation. We can change it though

Reply:
Then I would remove it. Otherwise it feels like we are allowing the user to create an instance with a number that may not make sense.

Reply:
Sorry, I thought you were asking whether it made sense to have a new regular **public** constructor side by side with the named constructor.

The existing regular **private** constructor is not usable by anyone outside the class. It is hidden behind the named constructor, so I do think it is correct to have it this way...